CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
+your Python code, your experience and what you can contribute is
+important to the project's success.
+
+
+Getting started, building, and testing
+--------------------------------------
+
+If you haven't already, take a look at the project's README file and
+the [Mypy documentation](http://mypy.readthedocs.org/en/latest/), and
+try adding type annotations to your file and type-checking it with Mypy.
+
+
+Discussion
+----------
+
+If you've run into behavior in Mypy you don't understand, or you're
+having trouble working out a good way to apply it to your code, or
+you've found a bug or would like a feature it doesn't have, we want to
+hear from you!
+
+Our main forum for discussion is the project's [GitHub issue
+tracker](https://github.com/python/mypy/issues).  This is the right
+place to start a discussion of any of the above or most any other
+topic concerning the project.
+
+We have an IRC channel, `#python-mypy` on irc.freenode.net.  This is
+lightly used, but some Mypy core developers are almost always present;
+feel free to find us there and we're happy to chat.  Substantive
+technical discussion will be directed to the issue tracker.
+
+#### Code of Conduct
+
+Everyone participating in the Mypy community, and in particular in our
+issue tracker, pull requests, and IRC channel, is expected to treat
+other people with respect and more generally to follow the guidelines
```

Comment:
This was the graf I found trickiest to word -- it's tricky partly because this is the one spot where some people may be motivated to take any ambiguity to give the text a perverse reading, so I want to cut off every avenue I can to reading it wrong. And I also didn't want to make it super cumbersome or verbose. :-)

I wouldn't want to just remove "more generally", because then it reads as if "treat other people with respect" and "follow the Python CoC" are independent things, as if the CoC isn't about treating people with respect. Definitely open to alternate suggestions.